charts/timed/templates/service-postgresql.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.postgresql.ingress.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: postgresql-ingress
+  labels:
+    {{- include "timed.labels" . | nindent 4 }}
+    app: postgresql
```

Comment:
Can't this be done with a [common label](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)? (ie. one with a app.kubernetes.io prefix)

Otherwise we should use something with a `k8s.adfinis.com` prefix.